packages/sx05re/emulationstation-addon/package.mk

```diff
@@ -13,7 +13,7 @@ PKG_ARCH="any"
 PKG_LICENSE="GPL"
 PKG_SITE="https://github.com/shantigilbert/EmulationStation"
 PKG_URL="$PKG_SITE.git"
-PKG_DEPENDS_TARGET="toolchain SDL2-git freetype curl freeimage vlc bash rapidjson ${OPENGLES}"
+PKG_DEPENDS_TARGET="toolchain SDL2-git freetype curl freeimage vlc bash rapidjson ${OPENGLES} gcc"
```

Comment:
Weird that this one is needed, since gcc should be built by toolchain...

Reply:
Toolchain builds gcc for host, but I understand target libgomp was needed.